tests/tasks/test_example.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from openedx_webhooks.tasks.example import add
+
+
+def test_add():
+    assert add(5, 6) == 11
+
+
+if __name__ == '__main__':
```

Comment:
No, I can take it out. The idea is that you can just execute the file itself and the test will run.